client/app/components/composites/AvatarDropdown/AvatarDropdown.js

```diff
@@ -32,8 +32,12 @@ class AvatarDropdown extends Component {
     }
   }
 
-  handleBlur() {
-    this.setState({ isOpen: false });// eslint-disable-line react/no-set-state
+  handleBlur(event) {
+    // FocusEvent is fired faster than the link elements native click handler
```

Comment:
"The FocusEvent interface represents focus-related events like focus, blur, focusin, or focusout."